frontend/components/PrivacyView.vue

```diff
@@ -0,0 +1,205 @@
+                        you a small fee for this service.
+                    </p>
+                    <p>
+                        The right to rectification – You have the right to request that we correct any information you
+                        believe is inaccurate. You also have the right to request that we complete the information you
+                        believe is incomplete.
+                    </p>
+                    <p>
+                        The right to erasure – You have the right to request that we erase your personal data, under
+                        certain conditions.
+                    </p>
+                    <p>
+                        The right to restrict processing – You have the right to request that we restrict the
+                        processing of your personal data, under certain conditions.
+                    </p>
+                    <p>
+                        The right to object to processing – You have the right to object to our processing of your
+                        personal data, under certain conditions.
+                    </p>
+                    <p>
+                        The right to data portability – You have the right to request that we transfer the data that we
+                        have collected to another organization, or directly to you, under certain conditions.
+                    </p>
+                    <p>
+                        If you make a request, we have one month to respond to you. If you would like to exercise any
+                        of these rights, please contact us.
+                    </p>
+                </TextBlock>
+                <TextBlock title="Children's Information" class="TextBlock">
+                    <p>
+                        Another part of our priority is adding protection for children while using the internet. We
+                        encourage parents and guardians to observe, participate in, and/or monitor and guide their
+                        online activity.
+                    </p>
+                    <p>
+                        Maker Protocol - Unified Auctions does not knowingly collect any Personal Identifiable
+                        Information from children under the age of 13. If you think that your child provided this kind
+                        of information on our website, we strongly encourage you to contact us immediately and we will
+                        do our best efforts to promptly remove such information from our records.
+                    </p>
+                </TextBlock>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import TextBlock from '~/components/common/TextBlock.vue';
+
+export default Vue.extend({
+    components: {
+        TextBlock,
+    },
+});
+</script>
+
+<style scoped>
+.TextBlock {
+    @apply flex-col space-y-2;
+}
```

Comment:
`flex-col` can not be applied without `flex`. I also don't think sure that applying it to the `TextBlock` component will apply it the default slot wrapper instead of the component's root element (or in case the structure of the `TextBlock` changes, those styles will be lost). You can instead directly apply styling via `p + p` selector